core/templates/dev/head/components/forms/image_uploader_directive.html

```diff
@@ -1,38 +1,101 @@
 <div class="image-uploader-drop-area">
   <div class="image-uploader-text" translate="I18N_DIRECTIVES_DRAG_IMAGE_HERE"></div>
-</div>
-
-<div class="image-uploader-file-input-container">
-  <span translate="I18N_DIRECTIVES_OR_SELECT_FILE"></span>
-  <input type="file" class="image-uploader-file-input" ng-class="fileInputClassName">
+  <br>
```

Comment:
Can we do this using margins/padding instead? Using br is a bit of an antipattern in this context.